doc/guides/test/sanitycheck.rst

```diff
@@ -162,9 +162,35 @@ Test Cases
 
 Test cases are detected by the presence of a 'testcase.yaml' or a 'sample.yaml'
 files in the application's project directory. This file may contain one or more
-entries in the test section each identifying a test scenario. The name of
-the test case only needs to be unique for the test cases specified in
-that testcase meta-data.
+entries in the test section each identifying a test scenario.
+
+The name of each testcase needs to be unique in the context of the overall
+testsuite and has to foolow basic rules:
+
+#. The format of the test identifier shall be a string without any spaces or
+   special charachters consisting of multiple sections delimited with a dot (.).
```

Comment:
What are "special characters"?  Are only (lowercase) alpha-numeric characters and underscore allowed?